README.md

```diff
@@ -16,7 +16,7 @@ To install Thyra, please follow the instructions available in the [Installation
 
 ## Install a plugin
 
-Copy/paste the latest release .zip file to Thyra [plugin manager page](https://my.massa/thyra/plugin-manager/)
+Copy/paste the latest release .zip file to Thyra [plugin manager page](https://station.massa/thyra/plugin-manager/)
```

Comment:
@Thykof there is an intersection of other endpoints using same route : `/plugin-manager` first one for front-end and second one for api calls , since there is enough changes in this PR , i prefer not tackle the breaking change also in swagger for plugin-manager and do it in seperate PR after deciding how we will handle api routes , i suggest to use something like `/plugin-manager/api` and `/plugin-manager` for front-end

Reply:
okay